packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../../devtools.dart' as devtools;
+import '../../flutter/common_widgets.dart';
+import '../../flutter/screen.dart';
+import '../../version.dart';
+import '../info_controller.dart';
+
+class InfoScreen extends Screen {
+  const InfoScreen() : super('Info');
+
+  @override
+  Widget build(BuildContext context) => InfoScreenBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return Tab(
+      icon: Icon(Icons.info),
+      text: 'Info',
+    );
+  }
+
+  /// The key to identify the flag list view
+  @visibleForTesting
+  static const Key flagListKey = Key('Info Screen Flag List');
+
+  /// The key to identify the flutter version view.
+  @visibleForTesting
+  static const Key flutterVersionKey = Key('Info Screen Flutter Version');
+}
+
+class InfoScreenBody extends StatefulWidget {
+  @override
+  _InfoScreenBodyState createState() => _InfoScreenBodyState();
+}
+
+class _InfoScreenBodyState extends State<InfoScreenBody> {
+  FlutterVersion _flutterVersion;
+  FlagList _flagList;
+
+  @override
+  void initState() {
+    super.initState();
+    InfoController(
+      onFlagListChanged: (flagList) => setState(() {
+        _flagList = flagList;
+      }),
+      onFlutterVersionChanged: (flutterVersion) => setState(() {
+        _flutterVersion = flutterVersion;
+      }),
+    )..entering();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          'Version Information',
+          style: textTheme.headline,
+        ),
+        const PaddedDivider(),
+        if (_flutterVersion != null) _VersionInformation(_flutterVersion),
+        const Padding(padding: EdgeInsets.only(top: 16.0)),
```

Comment:
should we define 16 and 4 as constants in a file under the src/flutter directory so we can tweak the app to have more or less whitespace without having to change a ton of places.

Reply:
I think that's a good refactoring for a follow-up PR.